### PR TITLE
DOC-1451 - RS: Update Create Replica Of database article

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1451/rs/databases/import-export/replica-of/create/).

File: content/embeds/tls-configuration-procedure.md [hunk 1]
```diff
@@ -1,54 +1,62 @@
...
+    1. Select the **Add** button to open the certificate dialog.  
+    ![icon_add](/images/rs/icon_add.png#no-click "Add button") 
 
         ![database-tls-replica-certs](/images/rs/database-tls-replica-certs.png
         "Database TLS Configuration")
 
-    1. Paste the text of the certificates in the box.
-    1. Click ![icon_save](/images/rs/icon_save.png#no-click "Save")
-    to save the certificates.
+    1. Paste the certificate text into the text box below the **Enforce client authentication** checkbox.
```
Comment (rrelledge) [inline content/embeds/tls-configuration-procedure.md:40]:
```suggestion
    1. Enter the copied certificate text into the text box below the **Enforce client authentication** checkbox.
```

File: content/embeds/tls-configuration-procedure.md [hunk 2]
```diff
@@ -58,22 +66,24 @@ To enable TLS for Replica Of and client communication on the source database:
     The certificates of the clients that connect to the database.
```
Comment (rrelledge) [inline content/embeds/tls-configuration-procedure.md:66]: Just a suggestion to fix this sentence a bit. Or maybe we could present the two required types of certificates (syncer certificates and client certificates) as bullet points instead.
```suggestion
    You also need to add the certificates of the clients that connect to the database.
```

File: content/embeds/tls-configuration-procedure.md [hunk 3]
```diff
@@ -58,22 +66,24 @@ To enable TLS for Replica Of and client communication on the source database:
...
+    1. Use the **Add** button to open the certificate dialog.  
+    ![icon_add](/images/rs/icon_add.png#no-click "Add button")
 
         ![database-tls-replica-certs](/images/rs/database-tls-replica-certs.png
         "Database TLS Configuration")
 
-    1. Paste the text of the certificates in the box.
-    1. Click ![icon_save](/images/rs/icon_save.png#no-click "Save")
-    to save the certificates.
+    1. Paste the certificate text into the text box below the **Enforce client authentication** checkbox.
```
Comment (rrelledge) [inline content/embeds/tls-configuration-procedure.md:84]:
```suggestion
    1. Enter the copied certificate text into the text box below the **Enforce client authentication** checkbox.
```

File: content/rs/databases/import-export/replica-of/create.md [hunk 4]
```diff
@@ -8,118 +8,125 @@ categories: ["RS"]
...
-{{< /note >}}
+Your apps can connect to the source database to read and write data; they can also use any replica for read-only access.
 
-## Configuring Replica Of
+Replica Of can model a variety of data relationships, including:
+
+- One-to-many relationships, where multiple replicas copy a single source database.
+- Many-to-one relationships, where a single replica collects data from multiple source databases.
+
+When you change the replica status of a database, by adding, removing, or changing sources, the replica database is synchronized to the new sources.
```
Comment (rrelledge) [inline content/rs/databases/import-export/replica-of/create.md:36]:
```suggestion
When you change the replica status of a database by adding, removing, or changing sources, the replica database is synchronized to the new sources.  
```

File: content/rs/databases/import-export/replica-of/create.md [hunk 5]
```diff
@@ -8,118 +8,125 @@ categories: ["RS"]
...
-        1. In the destination database, paste the URL of the source endpoint in the **Replica Of** box, and click ![Save](/images/rs/icon_save.png#no-click "Save").
+            If you regenerate the password, replication to existing destinations fails until their credentials are updated with the new password.
 
-        {{< note >}}
-For a source database on a different RS cluster,
-you can [compress the replication data]({{< relref "/rs/databases/import-export/replica-of/#data-compression-for-replica-of">}}) to save bandwidth.
-        {{< /note >}}
+        1. In the destination database, paste the URL of the source endpoint to the **Replica Of** edit box.  
+        
+        Use the **Save** button to save your changes.
```
Comment (rrelledge) [inline content/rs/databases/import-export/replica-of/create.md:85]:
```suggestion
        1. Use the **Save** button to save your changes.  
```

File: content/rs/databases/import-export/replica-of/create.md [hunk 6]
```diff
@@ -8,118 +8,125 @@ categories: ["RS"]
...
 
             ```sh
-            redis://:<redis_password>@<hostname>:<database_port>
+            redis://:<password>@<host>:<port>
             ```
 
             Where the password is the Redis password represented with URL encoding escape characters.
 
-        - If the database has no password -
+        - For databases without passwords: -
```
Comment (rrelledge) [inline content/rs/databases/import-export/replica-of/create.md:100]:
```suggestion
        - For databases without passwords:
```

File: content/rs/databases/import-export/replica-of/create.md [hunk 7]
```diff
@@ -8,118 +8,125 @@ categories: ["RS"]
...
+## Configure TLS on replica database
 
 When you enable TLS for Replica Of, the Replica Of synchronization traffic uses TLS certificates to authenticate the communication between the source and destination clusters.
+
 To encrypt the Replica Of synchronization traffic, you must also [configure encryption for the source database](#configuring-encryption-of-replica-of-traffic-on-the-source-database).
 
 To enable TLS for Replica Of in the destination database:
 
-1. Hover over the URL of the source endpoint and click ![Unencrypted](/images/rs/icon_unlocked.png#no-click "Unencrypted"):
+1. Select the **Enable TLS Authentication** button
```
Comment (rrelledge) [inline content/rs/databases/import-export/replica-of/create.md:116]:
```suggestion
1. Select the **Enable TLS Authentication** button:
```

File: content/rs/databases/import-export/replica-of/create.md [hunk 8]
```diff
@@ -8,118 +8,125 @@ categories: ["RS"]
...
 
     ![Encrypt Replica-of](/images/rs/replicaof-unencrypted.png)
 
-1. From the admin console of the cluster that hosts the source database,
-    go to **settings** > **general** and copy the proxy certificate.
-1. Paste it as the **Source Cluster Certificate** for the destination database:
+1. From the admin console of the cluster hosting the source database,
+    select the **Settings** menu and then go to the **General** tab.  Copy the full text of the proxy certificate to the Clipboard.
+
+1. Paste the certificate text as the **Source Cluster Certificate** for the destination database:
```
Comment (rrelledge) [inline content/rs/databases/import-export/replica-of/create.md:124]:
```suggestion
1. Enter the copied certificate text as the **Source Cluster Certificate** for the destination database:
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a few small suggestions.

File: content/rs/databases/import-export/replica-of/create.md (hunk 8 above)
Comment (lanceleonard) [reply content/rs/databases/import-export/replica-of/create.md]: Fixed.

File: content/rs/databases/import-export/replica-of/create.md (hunk 7 above)
Comment (lanceleonard) [reply content/rs/databases/import-export/replica-of/create.md]: Updated, though I didn't use a colon (due to the colon on the previous line).

File: content/rs/databases/import-export/replica-of/create.md (hunk 6 above)
Comment (lanceleonard) [reply content/rs/databases/import-export/replica-of/create.md]: Fixed.

File: content/rs/databases/import-export/replica-of/create.md (hunk 5 above)
Comment (lanceleonard) [reply content/rs/databases/import-export/replica-of/create.md]: Fixed.

File: content/rs/databases/import-export/replica-of/create.md (hunk 4 above)
Comment (lanceleonard) [reply content/rs/databases/import-export/replica-of/create.md]: Fixed.

File: content/embeds/tls-configuration-procedure.md (hunk 3 above)
Comment (lanceleonard) [reply content/embeds/tls-configuration-procedure.md]: Fixed.

File: content/embeds/tls-configuration-procedure.md (hunk 2 above)
Comment (lanceleonard) [reply content/embeds/tls-configuration-procedure.md]: Fixed.

File: content/embeds/tls-configuration-procedure.md (hunk 1 above)
Comment (lanceleonard) [reply content/embeds/tls-configuration-procedure.md]: Fixed.

File: content/embeds/tls-configuration-procedure.md [hunk 9]
```diff
@@ -1,54 +1,62 @@
...
+
     - For a new database - **Require TLS for Replica Of communications only** is
     selected by default.
+
     - For an existing database that is configured to **Require TLS for all
     communications** - Select **Require TLS for Replica Of communications only**.
 
-    By default, client authentication is enforced so you must enter the syncer certificates
-    of the clusters that host the destination databases.
+    By default, client authentication is enforced.  This means must enter the syncer certificates
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/embeds/tls-configuration-procedure.md]: Fixed.

**Pull request merged**